constraint/pkg/client/drivers/local/args.go

```diff
@@ -115,6 +120,23 @@ func DisableBuiltins(builtins ...string) Arg {
 	}
 }
 
+func AddClientTLSKeyPair(certFile, keyFile string) Arg {
+	return func(d *Driver) error {
+		d.clientCertFile = certFile
+		d.clientKeyFile = keyFile
+
+		return nil
+	}
+}
+
+func EnableExternalDataClientAuth() Arg {
+	return func(d *Driver) error {
```

Comment:
why do you need to return error if it's always going to be nil?

Reply:
I think this is to fit the generic `Arg` interface?